Dockerfile-cli

```diff
@@ -0,0 +1,16 @@
+FROM docker.io/library/python:3.11-slim-bullseye
+
+RUN addgroup --system nonroot \
+    && adduser --system --ingroup nonroot nonroot
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends git \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean
+
+WORKDIR /app
+
+RUN git clone -b master https://github.com/Clinical-Genomics/cg.git /app
```

Comment:
This won't work, we need to use the local files to pick up the changes in the feature branch